src/loaders/jobs/docker_img/Dockerfile.microtrait

```diff
@@ -0,0 +1,36 @@
+FROM mambaorg/micromamba:1.4.1
+
+# Note from micromamba - using an environment name other than "base" is not recommended
+ENV CONDA_ENV base
+
+COPY --chown=$MAMBA_USER:$MAMBA_USER /src/loaders/jobs/docker_img/env_microtrait.yaml /tmp/env_microtrait.yaml
+RUN micromamba install -y -n $CONDA_ENV -f /tmp/env_microtrait.yaml && \
+    micromamba clean --all --yes
+
+USER root
+ENV PATH=$PATH:/opt/conda/bin/
+
+# Install microTrait
+RUN Rscript -e 'install.packages("kmed", repos = "https://cloud.r-project.org/")' && \
+    Rscript -e 'devtools::install_github("jlw-ecoevo/gRodon")' && \
+    Rscript -e 'devtools::install_github("ukaraoz/microtrait")'
+
+# Setup of HMM database
+RUN Rscript -e 'library(microtrait)' && \
+    Rscript -e 'microtrait::prep.hmmmodels()' && \
+    Rscript -e 'list.files(file.path(.libPaths(), "microtrait/extdata/hmm/hmmpress"))'
```

Comment:
Is this something that should be installed outside of the dockerfile like the gtdb and checkm databases?

Reply:
the doc from microtrait didn't mention we could. It's not taking too much space either. I think it might not worth to read the R code and trying to figure out how hmm database is integrated with the microtrait tool. 
https://github.com/ukaraoz/microtrait/tree/master#5--setup-of-hmm-database

Reply:
If it's small then NBD

Reply:
:+1: